rudder-webapp/debian/dirs

```diff
@@ -1,6 +1,8 @@
 opt/rudder/bin
 opt/rudder/share/webapps
 opt/rudder/share/rudder-plugins
+opt/rudder/share/certificates
```

Comment:
It has been added to be consistent with the SPEC file (and it corrects a small mistake, this dir belongs to the rudder-webapp package and had been forgotten when we added cert management)